src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@ public interface MovieDao {
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
```

Comment:
Consider changing the return type of the `update` method to `void` or returning the updated `Movie` entity if applicable. Returning a `Boolean` might not be necessary if exceptions are used to indicate failure.